scripts/recan_bundle_tdr.py

```diff
@@ -396,7 +397,8 @@ def main(argv):
                            metadata_files=metadata)
 
     tdr_source = TDRSourceRef(id=args.source_id,
-                              spec=TDRSourceSpec(project='test_project',
+                              spec=TDRSourceSpec(prefix=Prefix.parse(''),
```

Comment:
I missed it because i was looking for usages of Spec.parse, not Prefix.parse. This is my fault.

Reply:
Also calling .effective here may also throw in an operators config.parition_prefix_length into the canned bundle, This might not be a problem the canned bundles are expected to be modified before they are committed. 

I recall in a past conversation, "I expect an emtpy string `''` to be insignificant", when talking talking about fields in a canned bundle.